packages/examples/packages/notifications/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "composite": false,
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "jsx": "react-jsxdev",
```

Comment:
Why did you add this? `jsxdev` has additional validation which we don't really need in Snaps, since JSX is validated by the client.